pyscriptjs/src/utils.ts

```diff
@@ -80,4 +79,24 @@ function handleFetchError(e: Error, singleFile: string) {
     showError(errorContent);
 }
 
-export { addClasses, removeClasses, getLastPath, ltrim, htmlDecode, guidGenerator, showError, handleFetchError };
+function getNamespace(name: string, runtime: any) {
+    if (name == 'DEFAULT_NAMESPACE') {
+        console.log('No pys-namespace attribute found, using default(global) namespace');
```

Comment:
Are you sure you want to log something here? I imagine users would still use the default namespace mostly, in which case this will spam the logs somewhat.

Reply:
Fair point - I'll remove it.